src/utils.py

```diff
@@ -0,0 +1,65 @@
+import mysql.connector
+
+
+def dbconnect(
+    hostname: str,
+    username: str,
+    dbname:str):
+    """
+    Method to connect database and python Script
+    
+    Parameters
+    ----------
+    hostname : (str) host name of mysql-server 
+    username : (str) username of mysql-server
+    dbname : (str) name of database to be accessed 
+    
+    Returns
+        Returns Tuple(con,cr) connection and cursor
+    """
+    
+    con = mysql.connector.connect(host=hostname,
+                                  user=username,
+                                  password = 'password',
+                                  database = dbname)
+
+    cr = con.cursor()
+    return con, cr
+
+
+def database(cr,query:str):
```

Comment:
remove query from the argument as this is fixed for this function. Instead provide database name as an argument.

inside the function, write the SQL query directly.

- Drop database if it exists
- then create database
- the function should return the databases lists

here's an example

```
def create_database(cursor: mysql.cursor.MySQLCursor, database_name: str) -> List:
    """
    Creates a new MySQL database with the given name.

    Parameters:
    -----------
    cursor : mysql.cursor.MySQLCursor
        The cursor object to use to execute SQL statements.
    database_name : str
        The name of the new database to create.

    Returns:
    --------
    None
    """
    # Drop the database if it already exists
    drop_db_query = f'DROP DATABASE IF EXISTS {database_name}'
    cursor.execute(drop_db_query)

    # Create the database
    create_db_query = f'CREATE DATABASE {database_name}'
    cursor.execute(create_db_query)

    # Show all databases on the MySQL server
    cursor.execute('SHOW DATABASES')
    databases = cursor.fetchall()
    return databases
```